scripts/gen_lazy_tensor.py

```diff
@@ -0,0 +1,95 @@
+from dataclasses import dataclass
+from typing import List, Union
+from torchgen.api.lazy import LazyIrSchema
+from torchgen.dest.lazy_ir import aten_symbol, GenLazyIR
+from torchgen.gen_lazy_tensor import run_gen_lazy_tensor
+from torchgen.model import NativeFunction, NativeFunctionsGroup
+from torchgen.api.types import (
+    BaseCType,
+    OptionalCType,
+    VectorCType,
+    kernel_signature,
+)
+import pathlib
+
+# assuming this file is under pytorch/xla
+torch_xla_root = pathlib.Path(__file__).parent.absolute().parent
+torch_root = torch_xla_root.parent
+aten_path = str(torch_root / "aten" / "src" / "ATen")
+shape_inference_hdr = str(torch_root / "torch" / "csrc" / "lazy" / "core" /
+                          "shape_inference.h")
+impl_path = str(torch_xla_root / "torch_xla" / "csrc" / "aten_xla_type.cpp")
+source_yaml = str(torch_xla_root / "xla_native_functions.yaml")
+
+
+@dataclass(frozen=True)
+class GenXlaLazyIR(GenLazyIR):
+
+  def lowering_function(self, f: Union[NativeFunctionsGroup,
+                                       NativeFunction]) -> str:
+    return f"""torch_xla::XlaOpVector Lower(LoweringContext* loctx) const override;"""
+
+  def node_ctor_args_call(self, all_args):
+    args_with_type = []
+    for i in all_args:
+      cpp_type = i.lazy_type.cpp_type()
+      if cpp_type == 'torch::lazy::Value':
```

Comment:
@wconstab This is a bit ugly but I am ok with it if it is the only one I need to repalce.

Reply:
oh, we should probably be able to make this cleaner. i will make a note to follow up, but i won't do anything before we sync up bc i'm not sure if it would disrupt you..

Reply:
this might also depend- how long do you think you want to use XlaValue instead of lazy::Value?  If its short term, maybe we just live with this for now.  if not, i think the cleaner fix would be to make use of the existing hook
`lazy_value_class="torch_xla::XlaValue",`
to change behavior inside LazyIrSchema and make it so 'lazy::value' is changed to XlaValue everywhere cleanly.

Reply:
I think I would need `torch_xla::XlaValue` until I remove `xla::shape` from our IR, so it might be a medium to long term thing.